pkgs/development/ocaml-modules/conduit/mirage.nix

```diff
@@ -0,0 +1,20 @@
+{ buildDunePackage, conduit-lwt
+, ppx_sexp_conv, sexplib, cstruct, mirage-stack, mirage-flow
+, mirage-flow-combinators, mirage-random, mirage-time, mirage-clock
+, dns-client, vchan, xenstore, tls, tls-mirage, ipaddr, ipaddr-sexp
+}:
+
+buildDunePackage {
+  pname = "conduit-mirage";
+
+  inherit (conduit-lwt) version src minimumOCamlVersion useDune2;
+
+  nativeBuildInputs = [ ppx_sexp_conv ];
+
+  propagatedBuildInputs = [
+    sexplib cstruct mirage-stack mirage-clock mirage-flow
+    mirage-flow-combinators mirage-random mirage-time
+    dns-client conduit-lwt vchan xenstore tls tls-mirage
+    ipaddr ipaddr-sexp
+  ];
+}
```

Comment:
There is no `meta` attribute. Is this on purpose?

Reply:
Was a mistake, fixed.